app/controllers/keywords_controller.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
 class KeywordsController < ApplicationController
-  def index; end
+  include Pagy::Backend
+
+  def index
+    pagy, keywords = pagy(KeywordsQuery.new(current_user).call)
```

Comment:
How about separate the KeywordQuery stuff into a separate method like we did in our [previous project](https://github.com/nimblehq/the1-web/blob/f97fcb48e50e605a92b35398dbd8e3ec6324a990/engines/the1_article/app/controllers/the1_article/articles_controller.rb#L15). I will be clean

```suggestion
    pagy, keywords = pagy(keywords).call)
```


```Ruby
private

def keywords
    pagy, keywords = pagy(KeywordsQuery.new(current_user).call)

   KeywordsQuery.new(current_user).call
end
```

Reply:
Nice!! I'll do that :)

Reply:
Fixed ;-)